app/controllers/static_pages_controller.rb

```diff
@@ -45,8 +45,8 @@ def relevant_portal_cookies
   #    GET /support
   #
   def support
-    @zones = CleanAirZone.visible_cazes
-    @temporary_exempt_cazes = CleanAirZone.temporarily_exempt_cazes
+    @zones = CleanAirZone.all.select(&:visible?)
+    @temporary_exempt_cazes = @zones.reject { |caz| caz.temporary_exemption_url.nil? }
```

Comment:
maybe there is a reason why you are doing it that way, but maybe wa can just select?

```suggestion
    @temporary_exempt_cazes = @zones.select(&:temporary_exemption_url)
```